api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -314,6 +318,24 @@ public Workspace updateWorkspaceAcls(
       }
       throw new BadRequestException(usersNotFound);
     }
+
+    // Finally, keep OWNER and billing project users in lock-step. In Rawls, OWNER does not grant
+    // canCompute on the workspace / billing project, nor does it grant the ability to grant
+    // canCompute to other users.
+    for (String user : Sets.union(updatedAclsMap.keySet(), aclsMap.keySet())) {
```

Comment:
Nit: Can we make this `String username`? Just because I tend to associate user with an object.

Reply:
Changed to `email` for consistency with the rest of the method.